globus_sdk/authorizers/refresh_token.py

```diff
@@ -103,7 +89,7 @@ def _get_new_access_token(self):
         """
         token_response = self.auth_client.oauth2_refresh_token(
             self.refresh_token)
-        self._set_expiration_time(token_response.expires_in)
+        self._set_expiration_time(token_response.expires_at)
```

Comment:
Maybe this is the counterargument on the top-level token response properties. We know that there's only one token in the response, and this reads rather nicely.

If we change that around a bunch, we need to remember to fix these as well.